static/app/views/performance/newTraceDetails/traceDrawer/details/span/components/spanSummaryLink.tsx

```diff
@@ -0,0 +1,102 @@
+import styled from '@emotion/styled';
+
+import type {SpanType} from 'sentry/components/events/interfaces/spans/types';
+import Link from 'sentry/components/links/link';
+import {IconStats} from 'sentry/icons';
+import {t, tct} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import type {EventTransaction} from 'sentry/types/event';
+import type {Organization} from 'sentry/types/organization';
+import {trackAnalytics} from 'sentry/utils/analytics';
+import {useLocation} from 'sentry/utils/useLocation';
+import {DATA_TYPE} from 'sentry/views/insights/browser/resources/settings';
+import {resolveSpanModule} from 'sentry/views/insights/common/utils/resolveSpanModule';
+import {useModuleURL} from 'sentry/views/insights/common/utils/useModuleURL';
+import {ModuleName} from 'sentry/views/insights/types';
+import {
+  querySummaryRouteWithQuery,
+  resourceSummaryRouteWithQuery,
+} from 'sentry/views/performance/transactionSummary/transactionSpans/spanDetails/utils';
+
+interface Props {
+  event: Readonly<EventTransaction>;
+  organization: Organization;
+  span: SpanType;
+}
+
+function SpanSummaryLink(props: Props) {
+  const location = useLocation();
+  const resourceBaseUrl = useModuleURL(ModuleName.RESOURCE);
+  const queryBaseUrl = useModuleURL(ModuleName.DB);
```

Comment:
These links appear to not work in some cases:

- Navigating from the Frontend Performance view > viewing a trace > clicking a DB span in the trace and clicking `View Query Summary`. I believe this is happening because `useModuleURL` does not include the `frontend` or `backend` slug. [Example](https://sentry.sentry.io/insights/frontend/trace/00bc62d08dc8430380778def6e8b4540/?fov=0%2C504&node=span-b82e3484fe1d7232&node=txn-2036dff2449e402c8d6913e41f7576f3&project=11276&query=%28%20transaction.op%3Apageload%20OR%20transaction.op%3Anavigation%20OR%20transaction.op%3Aui.render%20OR%20transaction.op%3Ainteraction%20%29&source=performance_transaction_summary&statsPeriod=24h&tab=events&timestamp=1732912723&transaction=%2Fsettings%2Forganization%2F)
- Same thing, but for asset summary when viewing a trace that you navigated to from the Backend view

But these problems are not directly related to this PR and can be fixed separately

Reply:
@DominikB2014 iirc you had some PR out to fix this. Just making sure you are aware of this.